WTWSMS/events/religious_events.txt

```diff
@@ -191,6 +199,7 @@ character_event = {
 		OR = {
 			religion = catholic
 			religion = fraticelli
+			religion = nicene
```

Comment:
The Nicene Patriarch is obviously going to complain about heresy, especially given how rampant it was in this time period.